index.js

```diff
@@ -881,12 +881,12 @@ RingPop.prototype.handleOrProxyAll =
                     url: req && req.url,
                     dest: dest
                 });
-                self.proxyReq({
+                self.proxyReq(_.defaults({
                     keys: keys,
                     req: req,
```

Comment:
yea, this was kind of the lazy approach. I dont want to introspect `opts` internals to the point that I would leave something out and plucking the ones out that you know are already in `opts` will be taken care of by `_.defaults()`